### DISCUSSION TITLE
Mau tanya ini kakak mengapa adblock plus tidak suport anti adblock killer
Pakai tampermonkey juga gk bekerja
Seperti di link ini contohnya
https://layardrama21.xyz/

### DISCUSSION THREAD

**Discussion by Jokopentil** [ABPindo/indonesianadblockrules#403] [Q&A] [answered]
![20230625_193009_rmscr.jpg](https: //github.com/ABPindo/indonesianadblockrules/assets/114223791/03d5b622-870a-49c4-a4a2-c085a7cd5102)
edit because NSFW

**Comment by partingscientist** [comment, marked as answer]
You might want to report the issue in https://github.com/abp-filters/abp-filters-anti-cv.

**Comment by Jokopentil** [comment]
How to i subscrbe this filter on android browser

**Reply by partingscientist** [reply]
This filter list seems to be enabled in the default installation, at least testing on the desktop extension. It's called "ABP filters (ABP Anti-Circumvention Filter List)". You have to report the issue there first though.

**Comment by Jokopentil** [comment]
Please screenshot
